test/e2e/mnist_pytorch_mcad_job_test.go

```diff
@@ -77,15 +77,23 @@ func TestMNISTPyTorchMCAD(t *testing.T) {
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{
 						{
-							Name:    "job",
-							Image:   GetPyTorchImage(),
+							Name:  "job",
+							Image: GetPyTorchImage(),
+							Env: []corev1.EnvVar{
+								corev1.EnvVar{Name: "PYTHONUSERBASE", Value: "/test2"},
+							},
 							Command: []string{"/bin/sh", "-c", "pip install -r /test/requirements.txt && torchrun /test/mnist.py"},
 							VolumeMounts: []corev1.VolumeMount{
 								{
 									Name:      "test",
 									MountPath: "/test",
 								},
+								{
+									Name:      "test2",
+									MountPath: "/test2",
+								},
 							},
+							WorkingDir: "/test2",
```

Comment:
Can we find a better name for the volume and the directory, maybe `workdir`?

Reply:
Thanks @astefanutti , I'll update this :)